src/Dropdown/Dropdown.tsx

```diff
@@ -48,46 +46,52 @@ export interface DropdownProps extends BlockProps {
   useTriggerComponentWidth?: boolean
   onOutsideClick?: (...rest: any) => void
   test?: string
+  makeCSS?: (position: Positioning, targetDimensions: Dimensions, parentDimensions: Dimensions) => string
 }
 
 export const Dropdown: React.FC<DropdownProps> = ({
   className,
   triggerComponent,
   children,
   open,
-  horizontal = 'auto',
-  vertical = 'auto',
+  horizontal = Horizontal.AUTO,
+  vertical = Vertical.AUTO,
   breakDistance = 20,
   useTriggerComponentWidth,
   onOutsideClick,
   test = 'dropdown',
+  makeCSS = makeDefaultCSS,
   ...blockProps
 }) => {
-  const [refs, layout, handleLayout] = useSnapEdgeToParent(breakDistance, { horizontal, vertical })
+  const [{ target , parent }, positioning, handleLayout] = useSnapEdgeToParent(breakDistance, { horizontal, vertical })
 
   useEffect(() => {
     window.addEventListener('resize', handleLayout)
     return () => window.removeEventListener('resize', handleLayout)
   }, [])
```

Comment:
Should we move this into useSnapEdgeToParent or should we leave it configurable when the position is recalculated @Michiel87

Reply:
Yes, it makes more sense to move it there since its coupled anyway